README.md

```diff
@@ -26,26 +26,35 @@ kubectl -n kube-system get deployment coredns -o yaml | \
 ```
 
 ### OpenShift
+> #### Requirements
```

Comment:
Remove the `>` so it gets rendered

Reply:
It was just to avoid opening a N+1 level header without any other headers on that level. Trying with a table.

Reply:
+1 to the tables. I think that looks better.